foundationdb/assets/configuration/spec.yaml

```diff
@@ -8,3 +8,34 @@ files:
   - template: instances
     options:
     - template: instances/default
+    - name: base_command
+      description: A base command to execute various queries on the FoundationDB instance
```

Comment:
Tweak it to:

The command to run in order to execute a query using the FoundationDB CLI. Typically this will invoke a locally installed `fdbcli` executable, however may be tweaked to run it inside of a container if a containerized deployment is being used and the Datadog agent runs outside of the container.